app/src/main/AndroidManifest.xml

```diff
@@ -74,28 +74,33 @@
         <activity
             android:name=".Startup_Pages.login_page"
             android:label="IdeaMeter"
-            android:theme="@style/AppTheme.NoActionBar">
-        </activity>
+            android:theme="@style/AppTheme.NoActionBar"></activity>
         <!--
              ATTENTION: This was auto-generated to add Google Play services to your project for
              App Indexing.  See https://g.co/AppIndexing/AndroidStudio for more information.
         -->
         <meta-data
             android:name="com.google.android.gms.version"
             android:value="@integer/google_play_services_version" />
+
         <activity
             android:name=".Startup_Pages.about_idealojy"
             android:label="About Idealojy"
             android:theme="@style/AppTheme.NoActionBar" />
-
-        <activity android:name=".Startup_Pages.splash"
+        <activity
+            android:name=".Startup_Pages.splash"
```

Comment:
Line number:93 All the strings in the xml files will be defined and accessed from res/values/strings.xml file.
Like this line should have been like this.
@OmerMajeed correct all these changes

- [ ] All string values referred from strings.xml

android:label=@string/app_name